apps/services/sessions/src/app/sessions/sessions-result.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger'
+
+import { PageInfo } from '@island.is/nest/pagination'
+
+import { Session } from './session.model'
+
+export class SessionsResultDto {
+  @ApiProperty({ required: true })
+  totalCount!: number
+
+  @ApiProperty({ type: () => [Session], required: true })
```

Comment:
```suggestion
  @ApiProperty({ type: [Session] })
```
The type property don't need to be lazy loaded with a function, unless there is a circular dependency.

Reply:
Fixed.